### PR TITLE
Default gradient

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

File: R/thematic.R [hunk 1]
```diff
@@ -461,8 +462,8 @@ sequential_gradient <- function(fg_weight = 0.75, bg_weight = 0.5, fg_low = TRUE
   if (n < 3) {
     stop("`n` must be 3 or more.", call. = FALSE)
   }
-  if (!is.logical(fg_low)) {
-    stop("`fg_low` must be `TRUE` or `FALSE`", call. = FALSE)
+  if (!is.logical(bg_high) && !identical(bg_high, "auto")) {
```
Comment (schloerke) [inline R/thematic.R:465]: Easier to read
```suggestion
  if (!(is.logical(bg_high) || identical(bg_high, "auto"))) {
```

File: R/thematic.R [hunk 2]
```diff
@@ -461,8 +462,8 @@ sequential_gradient <- function(fg_weight = 0.75, bg_weight = 0.5, fg_low = TRUE
...
+    stop("`bg_high` must be either `'auto'`, `TRUE` or `FALSE`", call. = FALSE)
```
Comment (schloerke) [inline R/thematic.R:466]:
```suggestion
    stop("`bg_high` must be either `'auto'`, `TRUE`, or `FALSE`", call. = FALSE)
```
I subscribe to the oxford comma

File: R/thematic.R [hunk 3]
```diff
@@ -461,8 +462,8 @@ sequential_gradient <- function(fg_weight = 0.75, bg_weight = 0.5, fg_low = TRUE
...
   }
```
Comment (schloerke) [inline R/thematic.R:467]: There is a corner case with `NA` values being logical values.  Could instead call `isTRUE` and not throw.
```suggestion
  if (!identical("auto")) {
    bg_high <- isTRUE(bg_high)
  }
```

Comment (cpsievert) [reply R/thematic.R]: Oh, right --- I think I'd prefer to accept only those three values

**Pull request closed without merging**